src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Debug/ConsoleDebugLaunchProviderTests.cs

```diff
@@ -391,25 +391,7 @@ public void GetExeAndArgumentsWithEmptyArgs()
             var targets = await debugger.QueryDebugTargetsAsync(0, activeProfile);
 
             Assert.Single(targets);
-            Assert.Equal(@"C:\library.dll", targets[0].Executable);
```

Comment:
This will break Test Explorer debugging, this was a very deliberate change in https://github.com/dotnet/project-system/pull/3648. You should make this test pass.